onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -228,6 +228,21 @@ def run_with_iobinding(self, iobinding, run_options=None):
         """
         self._sess.run_with_iobinding(iobinding._iobinding, run_options)
 
+    def run_forward(self, iobinding, run_options):
```

Comment:
I actually think it might not be the best file to host the API. I will keep this comment open and revisit the naming and where to place the API after the discussion with inference team.